Client/Frontend/Browser/TabManager.swift

```diff
@@ -165,6 +165,11 @@ class TabManager: NSObject {
     private class func getNewConfiguration() -> WKWebViewConfiguration {
         let configuration = WKWebViewConfiguration()
         configuration.processPool = WKProcessPool()
+        // Setting the Unique applicationName for the user agent to identify Brave.
+        if #available(iOS 13.0, *) {
+            configuration.applicationNameForUserAgent = Preferences.General.alwaysRequestDesktopSite.value ?
+            UserAgent.desktopUAApplicationName() : UserAgent.mobileUAApplicationName()
```

Comment:
In both cases won't this value be: `FxiOS`?

Reply:
No, in mobile init will be FxiOS and for desktop it will be Safari

Reply:
This does not seem correct to me. Why is this? Is it something from FF upstream?